atm.js

```diff
@@ -1,21 +1,67 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function() {
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var cBal = 0;
+var sBal = 0;
 
-}
+$("#cDepositButton").on("click", function() {
+  var cAmt = parseInt($("#cplaceholder").val());
+  cBal = (cBal + cAmt);
+  $("#checkingBalance").text("$" + cBal)
+})
 
-function checkingWithdrawal(amount){
+$("#cWithdrawButton").on("click", function() {
+  var cAmt = parseInt($("#cplaceholder").val());
 
-}
+  if (cBal > cAmt) {
+  cBal = (cBal - cAmt);
+  $("#checkingBalance").text("$" + cBal);
+  }
+  else if (cBal === cAmt) {
+    cBal = cBal - cAmt;
+    $("#checkingBalance").text("$" + cBal);
 
-function savingsDeposit(amount){
+    alert("You can't have a zero balance. We'll transfer $1 from your savings because nobody likes negative numbers.")
+    cBal = 1
+    $("#checkingBalance").text("$" + cBal);
 
-}
+    sBal = sBal - 1;
+    $("#savingsBalance").text("$" + sBal);
+  }
 
-function savingsWithdrawal(amount){
+  else if (cBal < cAmt && sBal > cAmt ) {
+    var cDiff = (cAmt - cBal);
+    cBal = (cBal - cAmt);
+    $("#checkingBalance").text("$" + cBal);
 
-}
+    var sDiff = cDiff + 1
```

Comment:
I really like your variable naming convention. makes it very easy to read and understand your code.